src/components/footer/styles.tsx

```diff
@@ -1,5 +1,3 @@
 import styled from 'styled-components'
 
-const Footer = styled.footer``
-
-export { Footer }
+export const StyledFooter = styled.footer``
```

Comment:
This is unnecessary component, it can be replaced with just `<footer>...</footer>`

Reply:
Was replaced